thorvg/examples/thorvg-example/main/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+idf_component_register(
+    SRCS "display_main.c"
+    INCLUDE_DIRS "."
+    PRIV_REQUIRES spiffs)
+
+set_source_files_properties(
+    "display_main.c"
+    PROPERTIES COMPILE_OPTIONS
+    "-Wno-strict-prototypes")
```

Comment:
I wonder if we can remove this if we upgrade the thorvg to 0.13.6? See: https://github.com/thorvg/thorvg/compare/v0.13.5...v0.13.6 and https://github.com/thorvg/thorvg/commit/946abbddf2b396aea4fccee944e41fd5745db809

Reply:
> I wonder if we can remove this if we upgrade the thorvg to 0.13.6? See: [thorvg/thorvg@v0.13.5...v0.13.6](https://github.com/thorvg/thorvg/compare/v0.13.5...v0.13.6) and [thorvg/thorvg@946abbd](https://github.com/thorvg/thorvg/commit/946abbddf2b396aea4fccee944e41fd5745db809)

Yes, it fixed this warning, I update it now.

Reply:
when you upgrade the thorvg, don't forget to update the sbom as well